antarest/study/business/aggregator_management.py

```
@@ -196,18 +359,33 @@ def aggregate_output_data(self) -> pd.DataFrame:
         Aggregates the output data of a study and returns it as a DataFrame
         """
 
-        # Checks if mc-ind results exist
-        if not self.mc_ind_path.exists():
-            raise OutputNotFound(self.output_id)
+        if self.mc_root == MCRoot.MC_IND:
+            # check that the `mc_years` is Sequence[int]
+            assert self.mc_years is not None, "mc_years should be a `Sequence` of integers"
+
+            # Checks if mc-ind results exist
+            if not self.mc_ind_path.exists():
+                raise OutputNotFound(self.output_id)
```

It's not something you changed in this dev but I believe we should be more specific with these exceptions.
3 things can happen:
1- The `output` folder doesn't exist, we should raise an OutputNotFound exception
2- The `mc-ind` folder doesn't exist, we should specify that it's just this folder missing
3- The `mc-all` folder doesn't exist, same as 2-